pages/advanced-algorithms/available-algorithms/gnn_link_prediction.mdx

```diff
@@ -11,7 +11,7 @@ import GitHub from '/components/icons/GitHub'
 # link_prediction with GNN
 
 <Callout type="info">
-This algorithm is not available in MAGE 1.14 and newer versions.
+This algorithm is unavailable in MAGE versions 1.14 to 1.21 but is supported again starting from version 1.22.
```

Comment:
I think this could be formulated better, instead of negative -> positive do positive -> negative, but can't think of good way to frame it.

Reply:
Maybe this?

> Since we resolved issues with symbol loading collision between Memgraph and
modules, **this algorithm is available again**, starting with MAGE version 1.22.
The algorithm was unavailable in MAGE from version 1.14 to version 1.21.